test/hotspot/jtreg/vmTestbase/gc/gctests/LargeObjects/large001/large001.java

```diff
@@ -147,7 +151,7 @@ public void run() {
                     addObjRef(loadedClassInstance, loadedClass, depth, refs);
 
                     // Drop all references to the class and try to unload it
-                    Algorithms.eatMemory(getExecutionController());
+                    WhiteBox.getWhiteBox().fullGC();
```

Comment:
Just to be clear, this is now no different than the `unloader.unloadClass()` call below in terms of how it is implemented. I wonder why the author originally used eatMemory() here and unloadClass() below.

Reply:
I have changed the test always use WhiteBox.fullGC() instead of call unloadClass(). This test has been verified locally,